source/js/diva.js

```diff
@@ -153,14 +153,22 @@ class Diva
         }
         else
         {
+        	let requestHeaders =  {
```

Comment:
There is something strange going on with indents here. Could you please format this to follow the format of the rest of the code? (Spaces not tabs, 4-space indents)